src/pages/console/formbuilder/customize.mdx

```diff
@@ -1,13 +1,15 @@
 export const meta = {
-  title: "Customize form inputs",
-  description: "Use the Form Builder in Amplify Studio to customize React form components. You can add new form inputs, bind them to a field, customize labels, and add validation rules."
-}
+  title: 'Customize form inputs',
+  description:
+    'Use the Form Builder in Amplify Studio to customize React form components. You can add new form inputs, bind them to a field, customize labels, and add validation rules.'
+};
 
-Use the Form Builder in Amplify Studio to customize React form components. You can add new form inputs, bind them to a field, customize labels, and add validation rules.
+Use the Form Builder in [Amplify Studio](/console/adminui/start/#logging-in-and-creating-an-app) to customize React form components. You can add new form inputs, bind them to a field, customize labels, and add validation rules.
```

Comment:
Rather than linking to the login at the first mention of Studio, I recommend moving the link to the first instances of working inside Studio.

Here you might tell people to "To access form builder, log into Amplify Studio..." just above "the form builder interface consists of..." below